packages/fondue/src/components/FormField/FormField.cy.tsx

```diff
@@ -117,27 +117,44 @@ describe('Form Field Component', () => {
         cy.get(FORM_FIELD_TEXT_INPUT).focused();
     });
 
-    it('should render tooltip', () => {
-        cy.mount(COMPONENT_BASE_WITH_TOOLTIPS);
-        cy.get(FORM_FIELD_TOOLTIP_ONE_ICON).should('exist');
-    });
-
-    it('should render multiple tooltips', () => {
-        cy.mount(COMPONENT_BASE_WITH_TOOLTIPS);
-        cy.get(FORM_FIELD_LABEL).children().should('have.length', 2);
-        cy.get(FORM_FIELD_TOOLTIP_ONE_ICON).should('exist');
-        cy.get(FORM_FIELD_TOOLTIP_TWO_ICON).should('exist');
-    });
-
-    it('should open 1st tooltip dialog on hover', () => {
-        cy.mount(COMPONENT_BASE_WITH_TOOLTIPS);
-        cy.get(FORM_FIELD_TOOLTIP_ONE_ICON).trigger('mouseover');
-        cy.get(FORM_FIELD_TOOLTIP_ONE).should('exist');
-    });
-
-    it('should open 2nd tooltip dialog on hover', () => {
-        cy.mount(COMPONENT_BASE_WITH_TOOLTIPS);
-        cy.get(FORM_FIELD_TOOLTIP_TWO_ICON).trigger('mouseover');
-        cy.get(FORM_FIELD_TOOLTIP_TWO).should('exist');
+    describe('Tooltips', () => {
+        it('should render', () => {
+            cy.mount(COMPONENT_BASE_WITH_TOOLTIPS);
+            cy.get(FORM_FIELD_TOOLTIP_ONE_ICON).should('exist');
+        });
+
+        it('should render multiple', () => {
+            cy.mount(COMPONENT_BASE_WITH_TOOLTIPS);
+            cy.get(FORM_FIELD_LABEL).children().should('have.length', 2);
+            cy.get(FORM_FIELD_TOOLTIP_ONE_ICON).should('exist');
+            cy.get(FORM_FIELD_TOOLTIP_TWO_ICON).should('exist');
+        });
+
+        it('should open 1st tooltip dialog on hover', () => {
+            cy.mount(COMPONENT_BASE_WITH_TOOLTIPS);
+            cy.get(FORM_FIELD_TOOLTIP_ONE_ICON).trigger('pointerover', {
+                eventConstructor: 'MouseEvent',
+                pointerType: 'mouse',
+            });
+            cy.get(FORM_FIELD_TOOLTIP_ONE).should('exist');
+        });
+
+        it('should open 2nd tooltip dialog on hover', () => {
+            cy.mount(COMPONENT_BASE_WITH_TOOLTIPS);
+            cy.get(FORM_FIELD_TOOLTIP_TWO_ICON).trigger('pointerover', {
+                eventConstructor: 'MouseEvent',
+                pointerType: 'mouse',
+            });
+            cy.get(FORM_FIELD_TOOLTIP_TWO).should('exist');
+        });
+
+        it('should not open 1st tooltip on touch event', () => {
```

Comment:
Please remove these tests and add them in `packages/fondue/src/components/Tooltip/Tooltip.cy.tsx`

Reply:
I will leave the previous test inside, because they cover the Tooltips for this component and will add additional test to the Tooltip.

Reply:
Updated